src/qt/forms/sendcoinsdialog.ui

```diff
@@ -848,7 +848,7 @@
                  <item>
                   <widget class="QLabel" name="labelCustomPerKilobyte">
                    <property name="toolTip">
-                    <string>If the custom fee is set to 1000 satoshis and the transaction is only 250 bytes, then &quot;per kilobyte&quot; only pays 250 satoshis in fee, while &quot;total at least&quot; pays 1000 satoshis. For transactions bigger than a kilobyte both pay by kilobyte.</string>
+                    <string>For the transaction fee, you can specify the exact number of "satoshis per kilobyte (1,000 bytes)."  For transactions larger than 1 kilobyte, the fee will be calculated on a per-kilobyte basis.  For transactions smaller than 1 kilobyte, the fee will be apportioned according to the size of the transaction (e.g., for a transaction that is 500 bytes {half of 1 kilobyte}, a fee of 0.00000010 would therefore be reduced to 0.00000005).</string>
```

Comment:
There is no longer a distinction between less-than-1kB and other txes. You might as well remove that distinction altogether.

Reply:
@MarcoFalke Oh, so to clarify, that means that for transactions of 1 kilobyte or less that the fee would still be calculated on a per-kilobyte basis too then, and I should squash it down to the following only?

``For the transaction fee, you can specify the exact number of "satoshis per kilobyte (1,000 bytes)."``

Reply:
Yeah, it is always a fee rate (it just happens to be denominated in BTC/kB, and not satoshis/Byte)

Reply:
That's correct. But we already know it's for the transaction fee, and that you're specifying it. You aren't specifying satoshis per kB, you can specify it in BTC, mBTC, or uBTC. So you could just squash it down to "per kilobyte (1000 bytes)" - but that's almost what the label itself already says.

I prefer something like:

> Specify a custom fee per 1000 bytes of virtual transaction size."

Reply:
So all fee rates (set on the command line or the gui) are denominated in BTC/kB

Reply:
The gui allows me to specify the fee rate in BTC/kB, mBTC/kB, and uBTC/kB.

It's kind of unfortunate that we can't specify it in sat/B, since that seems to be emerging as a standard way to talk about fee rates.